Python/AudioBook/Audiobook.py

```diff
@@ -0,0 +1,12 @@
+import pyttsx3
+import PyPDF2
+book=open('software.pdf','rb')

```

Comment:
It would be great if rather than having hard-coded values, we can make use of Arguement Parsers to pass the File Name which can be read.